src/lwc/signals/core.ts

```diff
@@ -29,10 +29,12 @@ const UNSET = Symbol("UNSET");
 const COMPUTING = Symbol("COMPUTING");
 const ERRORED = Symbol("ERRORED");
 const READY = Symbol("READY");
+const MAX_STACK_DEPTH = 16;
 
 interface EffectNode {
   error: unknown;
   state: symbol;
+  stack_depth: number;
```

Comment:
Minor: Let's use camel case rather than snake case to keep it consistent with the other naming in this file